pytuya/__init__.py

```diff
@@ -121,6 +121,18 @@ def hex2bin(x):
     },
     "prefix": "000055aa00000000000000",    # Next byte is command byte ("hexByte") some zero padding, then length of remaining payload, i.e. command + suffix (unclear if multiple bytes used for length, zero padding implies could be more than one byte)
     "suffix": "000000000000aa55"
+  },
+  "bulb": {
```

Comment:
these are dupes for the outlet type, either update the dict entry for bulb with status after initialization... or remove completely, more on this below